CHANGELOG.rst

```diff
@@ -5,9 +5,12 @@
 
 *
 
-*
+* Add `strict` parameter to xfail marker. Thanks `@pquentin`_ for the
+  PR and `@nicoddemus`_ for noticing the issue.
```

Comment:
This is confusing, as it's only a doc fix, right? IMHO, I don't think this should be mentioned in the changelog at all.